### PR TITLE
Create consistent plot themes for baggr plots

### PR DESCRIPTION
**Pull request by be-green**
Add baggr_theme_get(), _set(), _replace(), and _update() methods, and apply them to all the places that the dev branch includes plotting methods. This does not yet impact the quantiles plotting method on the dev-quant branch.

### CODE REVIEW

File: R/plot.R [hunk 1]
```diff
@@ -14,3 +14,40 @@ plot.baggr <- function(x, ...) {
...
   do.call(baggr_plot, l)
 }
+
+#' Set, get, and replace themes for baggr plots
+#' @rdname baggr_theme_set
+#' @importFrom ggplot2 theme_grey
+#' @importFrom bayesplot theme_default
+#' @param new New theme to use for all baggr plots
+#' @description These functions get, set, and modify the ggplot2 themes
+#' of the baggr plots. By default, these match the bayesplot's package
```
Comment (wwiecek) [inline R/plot.R]: Should link to related documentation (in this case `bayesplot`, `bayesplot::bayesplot_theme_get` or `ggplot2::theme_get`), both via inline links where other pkgs are namechecked and in See Also.

File: R/plot.R [hunk 2]
```diff
@@ -14,3 +14,40 @@ plot.baggr <- function(x, ...) {
   l[["bg"]] <- x
   do.call(baggr_plot, l)
 }
+
+#' Set, get, and replace themes for baggr plots
+#' @rdname baggr_theme_set
+#' @importFrom ggplot2 theme_grey
+#' @importFrom bayesplot theme_default
+#' @param new New theme to use for all baggr plots
```
Comment (wwiecek) [inline R/plot.R]: must document `...` param ("A named list of theme settings."); otherwise fails R CMD CHECK

File: R/plot.R [hunk 3]
```diff
@@ -14,3 +14,40 @@ plot.baggr <- function(x, ...) {
...
+#' @description These functions get, set, and modify the ggplot2 themes
+#' of the baggr plots. By default, these match the bayesplot's package
+#' theme to make it easier to form cohesive graphs across this package
+#' and others.
+#' @return The get method returns the current theme, but all of the
+#' others invisibly return the old theme.
+#' @export
```
Comment (wwiecek) [inline R/plot.R]: Needs a couple of examples! I've never used `update` and `replace`, so am not sure. Again, for more details enough to link to other pkgs.

Comment (be-green) [reply R/plot.R]: I will add them and re-push.

File: R/plot.R (hunk 2 above)
Comment (be-green) [reply R/plot.R]: Thanks, not sure how I missed that.

File: R/plot.R (hunk 1 above)
Comment (be-green) [reply R/plot.R]: Added this as well.

**Comment by be-green** [comment]
Looks like the travis checks are failing because of convert_inputs for a quantiles model. I found this was breaking on my end too, but I'm not sure whether to try to fix it in this pull request or leave it for another time.

**Pull request merged**